shotover-proxy/src/transforms/protect/encrypt.rs

```diff
@@ -0,0 +1,66 @@
+use crate::message::MessageValue;
```

Comment:
Can we change the name of the file to `cryptography.rs` or something as it does both encryption and decryption operations

Reply:
good point, done